aws-s3-transfer-manager/tests/upload_checksum_test.rs

```diff
@@ -0,0 +1,829 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use std::{str::FromStr, task::Poll};
+
+use aws_s3_transfer_manager::{
+    io::{InputStream, PartData, PartStream, SizeHint},
+    operation::upload::{ChecksumStrategy, UploadOutput},
+    types::{ConcurrencySetting, PartSize},
+};
+use aws_sdk_s3::{
+    operation::{
+        complete_multipart_upload::CompleteMultipartUploadOutput,
+        create_multipart_upload::CreateMultipartUploadOutput, put_object::PutObjectOutput,
+        upload_part::UploadPartOutput,
+    },
+    types::{ChecksumAlgorithm, ChecksumType},
+};
+use aws_smithy_mocks_experimental::{mock, Rule, RuleMode};
+use aws_smithy_runtime::test_util::capture_test_logs::capture_test_logs;
+use bytes::Bytes;
+use pin_project_lite::pin_project;
+use test_common::mock_client_with_stubbed_http_client;
+
+const MEBIBYTE: usize = 1024 * 1024;
```

Comment:
We have some types for this already, we might want to be consistent and just use them ([example](https://github.com/awslabs/aws-s3-transfer-manager-rs/blob/dca33b430f4f89da6cc61c8b53f4d2232bc528f0/aws-s3-transfer-manager/src/config.rs#L15))

Reply:
Fixed, and fixed some other tests that were doing `const MEBIBYTE: ...`